### ISSUE TITLE
ValueError: cannot reshape array of size 756900 into shape (416,416,4)

### ISSUE THREAD

**Issue by ppp-one** [ckm3/Deep-Transit#11]
When running from the quick start:
`bboxes = dt.DeepTransit(lc, is_flat=True).transit_detection(model_path)`

I receive the following error:

```
C:\Users\peter\anaconda3\lib\site-packages\deep_transit\dt_lightcurve.py:317: UserWarning: The total number of progress bar is the upper limit.
  0%|          | 0/4 [00:00<?, ?it/s]
Loading Model: model_Kepler.pth
  0%|          | 0/4 [00:00<?, ?it/s]
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-4-efa83d90cf6f> in <module>
----> 1 bboxes = dt.DeepTransit(lc, is_flat=True).transit_detection(model_path)

~\anaconda3\lib\site-packages\deep_transit\dt_lightcurve.py in transit_detection(self, local_model_path, batch_size, confidence_threshold, nms_iou_threshold, device_str, backend)
    316         rough_length = math.ceil(math.ceil((len(self.lc) * exp_time - 30) / 25 + 1) * 4 / batch_size)
    317         warnings.warn('The total number of progress bar is the upper limit.')
--> 318         for data in tqdm(self._data_loader(batch_size=batch_size, backend=backend), total=rough_length):
    319             lc_data = data[:, 0]
    320             flux_min, flux_max = data[:, 2], data[:, 3]

~\anaconda3\lib\site-packages\tqdm\std.py in __iter__(self)
   1176 
   1177         try:
-> 1178             for obj in iterable:
   1179                 yield obj
   1180                 # Update and possibly print the progressbar.

~\anaconda3\lib\site-packages\deep_transit\dt_lightcurve.py in _data_loader(self, batch_size, backend)
    269             with warnings.catch_warnings():
    270                 warnings.simplefilter("ignore")
--> 271                 chunk = np.array(list(itertools.islice(it, batch_size)))
    272             if chunk.size == 0:
    273                 return

~\anaconda3\lib\site-packages\deep_transit\dt_lightcurve.py in _splited_lc_generator(self, backend)
    258                     if len(splited_flatten_lc) < 1 / exp_time:
    259                         continue
--> 260                     img_arr = _light_curve_to_image_array(splited_flatten_lc, (flux_min, flux_max))
    261                     image = np.array(Image.fromarray(img_arr).convert("L"))
    262                     image = backend.trans(image)

~\anaconda3\lib\site-packages\deep_transit\dt_lightcurve.py in _light_curve_to_image_array(lc_object, flux_range)
    118         ax.set_ylim([flux_range[0], flux_range[1]])
    119         fig.savefig(io_buf, format='raw', dpi=100)
--> 120         img_arr = np.reshape(np.frombuffer(io_buf.getvalue(), dtype=np.uint8),
    121                              newshape=(int(fig.bbox.bounds[3]), int(fig.bbox.bounds[2]), 4))
    122         io_buf.close()

<__array_function__ internals> in reshape(*args, **kwargs)

~\anaconda3\lib\site-packages\numpy\core\fromnumeric.py in reshape(a, newshape, order)
    297            [5, 6]])
    298     """
--> 299     return _wrapfunc(a, 'reshape', newshape, order=order)
    300 
    301 

~\anaconda3\lib\site-packages\numpy\core\fromnumeric.py in _wrapfunc(obj, method, *args, **kwds)
     56 
     57     try:
---> 58         return bound(*args, **kwds)
     59     except TypeError:
     60         # A TypeError occurs if the object does have such a method in its

ValueError: cannot reshape array of size 756900 into shape (416,416,4)
```

I am running on a cpu. Help?

**Comment by qq332982511** [comment]
you can provide the full code to replay this error

**Comment by qq332982511** [comment]
or you can try https://colab.research.google.com/drive/1f2tuvDRWacA2XDEqs1v6zEZxCOr7kib8?usp=sharing

**Comment by ppp-one** [comment]
> 
> 
> you can provide the full code to replay this error

The code is here: I was following the quickstart tutorial.

```
%matplotlib inline
import lightkurve as lk
import deep_transit as dt

model_path = 'model_Kepler.pth'

lc = lk.search_lightcurve('Kepler-1b', quarter=14, exptime='long').download();
lc = lc[lc.time.value>1340] # To save computating time
lc.plot()


bboxes = dt.DeepTransit(lc, is_flat=True).transit_detection(model_path)
```

**Comment by ckm3** [comment]
Could you provide your detailed OS and python environment info (pip list results or conda list results). We have tested it on some Linux systems but can't reproduce your error.

**Comment by ppp-one** [comment]
I think I have found the problem -- it's due to how I've set up my c.InlineBackend.rc for ipython_kernel_config/jupyter -- different figsize and dpi etc.  The code works fine outside of jupyter notebooks. 

Within jupyter I can run it like this instead:

```
import lightkurve as lk
import deep_transit as dt
import matplotlib as mpl
import matplotlib.pyplot as plt

model_path = 'model_Kepler.pth'

lc = lk.search_lightcurve('Kepler-1b', quarter=14, exptime='long').download();
lc = lc[lc.time.value>1340] # To save computating time

## this resets to mpl's default config
mpl.rcParams.update(mpl.rcParamsDefault) 

bboxes = dt.DeepTransit(lc, is_flat=True).transit_detection(model_path)

## the inline magic command resets it back to my custom config for plotting.
%matplotlib inline  
fig, ax = plt.subplots()
dt.plot_lc_with_bboxes(lc, bboxes, ax=ax, lw=1)
```

**Comment by ckm3** [comment]
Yes, you are right, ipython rcparams might overwrite our default plot settings, I will fix this issue later. Thank you.

**Comment by ckm3** [comment]
This issue has been fixed in 52f2e81b2beb0d73974741ea78ba84ea52e8fc2e
